tests/asgi/tests.py

```diff
@@ -194,6 +197,56 @@ async def test_post_body(self):
         self.assertEqual(response_body["type"], "http.response.body")
         self.assertEqual(response_body["body"], b"Echo!")
 
+    async def test_cancel_post_request_with_sync_processing(self):
+        loop = asyncio.get_event_loop()
+        # Events used to monitor the view processing from the parent test code
+        view_started_event = asyncio.Event()
+        view_finished_event = asyncio.Event()
+        # Record exception raised in the test view
+        exceptions = []
+
+        # This view will run in a new thread because it is wrapped in sync_to_async.
```

Comment:
Following the [coding style guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/), "Documentation, comments, and docstrings should be wrapped at 79 characters". Could you please re-wrap these?